Localization/cubature_kalman_filter/cubature_kalman_filter.py

```diff
@@ -0,0 +1,234 @@
+"""
+implementation of cubature kalman filter using CTRV motion model
```

Comment:
What does stand for CTRV?

Reply:
CTRV stands for Constant Turn Rate and Velocity motion model. It is a standard motion model used in kalman filters for the purpose of fusing sensor data from GPS and IMU and otainining an accurate estimate of position.
I referred these sources:
[http://fusion.isif.org/proceedings/fusion08CD/papers/1569107835.pdf](Comparison and Evaluation of Advanced
Motion Models for Vehicle Tracking)
[https://github.com/balzer82/Kalman/blob/master/Extended-Kalman-Filter-CTRV2.ipynb](Kalman by Balzer82)

Reply:
Could you please add the description and the link in the header comment?.

Reply:
Okay, I can do that.